### DISCUSSION TITLE
[Feature] Remove face from photo

### DISCUSSION THREAD

**Discussion by bearded-papa** [immich-app/immich#7818] [Feature Request]
### The feature

During face recognition, there are faces detected in photos in the background. It is not currently possible to remove these faces but you can only replace them with another.
Currently I am having to name these faces as 'Background Photo' to be able to remove them from other people.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #5962
